src/main/java/org/ethereum/beacon/discovery/network/NettyDiscoveryServerImpl.java

```diff
@@ -57,6 +60,7 @@ public NettyDiscoveryServerImpl(InetSocketAddress listenAddress) {
               @Override
               public void initChannel(NioDatagramChannel ch) {
                 ch.pipeline()
+                    .addFirst(new ChannelTrafficShapingHandler(0, trafficReadLimit))
```

Comment:
I'd suggest having a way to disable this entirely - e.g. by setting `trafficReadLimit` to 0.  I guess it could be set to Long.MAX_VALUE but would be nice to be able to skip adding the handler altogether.

Reply:
Done.